test/integration/model/upsert.test.js

```diff
@@ -248,6 +249,41 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         });
       });
 
+      it('does not overwrite createdAt time on update', function() {
+        var originalCreatedAt;
+        return this.User.create({ id: 42, username: 'john'}).bind(this).then(function() {
+          return this.User.findById(42);
+        }).then(function(user) {
+          originalCreatedAt = user.createdAt;
+          return this.sequelize.Promise.delay(1000).bind(this).then(function() {
```

Comment:
I must be misremembering then, I remember a lot of trouble last time I used these for anything other than extending timers. I'll experiment a bit tomorrow and see if I can get it to work.